packages/web/src/components/spa/HackAUBG/JudgesSection/JudgesSection.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import axios from 'axios';
+import { useState } from 'react';
+import { useEffect } from 'react';
+import { url } from '../../../../Global';
+import { JudgesCard } from './JudgesCard';
+import './judges_section.css';
+
+export const JudgesSection = () => {
+    const [jury, setJury] = useState([{}]);
+
+    const getJury = () => {
+        axios({
+            method: 'get',
+            url: url + '/api/jury'
+        })
+            .then((res) => {
+                setJury(res.data.data.data);
+            })
+            .catch((err) => {
+                console.log(err);
+            });
+    };
+
+    const handleClick = (event) => {
+        return () => {
+            window.open(event, '_blank', 'noopener noreferrer');
+        };
+    };
```

Comment:
const openNewTab = (url) => {window.open(...)}

As this is used in both sections, you could put the function in Global.js